### DISCUSSION TITLE
Tutorial Videos before Day 3 Live Session

### DISCUSSION THREAD

**Discussion by mahnoor-byte** [AammarTufail/python_ka_chilla2023#5] [General]
It was mentioned that videos will be uploaded after yesterday's live session. After that, we can discuss our queries in today's live session. I can't find the videos. Can anyone please share where I can find those?

**Comment by QuratUlAin946** [comment]
Dr. will share everything in the telegram group.

**Reply by mahnoor-byte** [reply]
Nothing there yet. That's why I asked.

**Reply by nouman073** [reply]
https://www.youtube.com/playlist?list=PL9XvIvvVL50Fba7psesg6ynQXdipw-yoN
playlist link,follow this link

**Reply by AammarTufail** [reply]
Uploaded and communicated

**Reply by mahnoor-byte** [reply]
Yes, thank you. Done with it.

**Comment by syedmoinuddinbukhari** [comment]
**_Simple Explained the PEMDAS and DMAS RULE_** 
![image](https://user-images.githubusercontent.com/119401213/210503469-df1dc29c-103e-4b5f-a217-1f8099f03af7.png)

**Comment by syedmoinuddinbukhari** [comment]
SIMPLE OPERATORS 
![image](https://user-images.githubusercontent.com/119401213/210504312-b73d4275-03ae-4893-b76c-6ce75bc35e08.png)

**Comment by syedmoinuddinbukhari** [comment]
Here is the Type Comparison 
![image](https://user-images.githubusercontent.com/119401213/210721814-46dbaa8a-51dc-44fa-83ee-22fa1b736fda.png)
